src/authorization/permissions/SparqlUpdateModesExtractor.ts

```diff
@@ -34,13 +35,14 @@ export class SparqlUpdateModesExtractor extends ModesExtractor {
     }
   }
 
-  public async handle({ body, target }: Operation): Promise<Set<AccessMode>> {
+  public async handle({ body, target }: Operation): Promise<AccessMap> {
     // Verified in `canHandle` call
     const update = (body as SparqlUpdatePatch).algebra as Algebra.DeleteInsert;
     const modes = new Set<AccessMode>();
+    const result = new IdentifierMap([[ target, modes ]]);
```

Comment:
```suggestion
    const requiredModes = new IdentifierMap([[ target, modes ]]);
```
(I'd avoid calling things `result` in general)

Reply:
![image](https://user-images.githubusercontent.com/3447363/177575103-34ce0ef9-fab6-43f6-a9aa-feea4a5cd236.png)
😇